### ISSUE TITLE
CPSV-AP: Identifier only mandatory for independent entities.

### ISSUE THREAD

**Issue by GeertThijs** [Informatievlaanderen/OSLO#26]
Identifier should only be mandatory for those entities which are independent of any other entity. In our opinion entities like Cost, Channel, ContacPoint , PeriodOfTime, Rule are properties (with a composite datatype) of PublicService rather than  autonomous entities. (See also previous issue.)

**Comment by GeertThijs** [comment]
Zie ook issue #

**Comment by dhaeneth** [comment]
@GeertThijs  zie ook https://joinup.ec.europa.eu/asset/cpsv-ap/issue/all-classes-are-blank-nodes-allowed

**Comment by GeertThijs** [comment]
Oké, dat is al iets. Mijn vraag gaat eigenlijk nog verder, nl moeten we er geen properties van maken tout court?

**Comment by dhaeneth** [comment]
Niet in rdf :-).

**Comment by GeertThijs** [comment]
Nee, dat weet ik intussen al wel. Maar ik ben hier een UML-model aan het maken, geen RDF-model.

**Comment by GeertThijs** [comment]
Intussen zijn Rule en Channel terug entities, zie laatste comment bij #7. De identifier issue blijft wel natuurlijk als onder Identifier een gepubliceerde identificator wordt verstaan, die gaan we pas kunnen geven als er een register of gelijkaardig voor de betrokken entiteit bestaat.

**Comment by DieterDePaepe** [comment]
[Joinup issue van Geert](https://joinup.ec.europa.eu/asset/cpsv-ap/issue/identifier-only-mandatory-independent-entities)

Properties tonen minder goed aan welke informatie samen hoort vind ik. Aparte entiteiten zijn duidelijker.
